### PR TITLE
Add satpam validation for required if not

### PR DESCRIPTION
**Pull request by sidneyrachel**
This is for application form validation

### CODE REVIEW

**Review by albertstevelino — CHANGES_REQUESTED** [review]
you forget to update the satpam package version

File: test/validators/required-if-not.spec.js [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+      it('should success if hasHome is No and address is defined', () => {
+        const result = validator.validate(simpleRules, {
+          hasHome: 'No',
+          address: 'somewhere over the rainbow'
+        });
+        const err = result.messages;
+
+        expect(result.success).to.equal(true);
+        expect(err).to.not.have.property('address');
+      });
```
Comment (albertstevelino) [inline test/validators/required-if-not.spec.js]: this test case is good, it is solid and meet the requirement, but your example somehow somewhat hurts my brain, address is required if has home is not yes (which is no). maybe you should switch the case.

Anyway, good work!

**Review by albertstevelino — APPROVED** [review]
minor update for test case example

File: test/validators/required-if-not.spec.js (hunk 1 above)
Comment (sidneyrachel) [reply test/validators/required-if-not.spec.js]: WKWK
done ya Ko (y)

**Pull request merged**